### PR TITLE
[Website] Ft Highlights Section Backend

### PR DESCRIPTION
**Pull request by FaithDaka**
#### Summary of Changes (What does this PR do?)
- Features the backend setup for the highlights section of the homepage.

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Status of maturity (all need to be checked before merging):

- [x] I consider this code done
- [x] I've tested this locally
- [ ] This branch has been LGTM'ed by a reviewer

#### How should this be manually tested?
* Fetch and checkout _ft-backend_ to run the updated admin portal.

#### What are the relevant tickets?
- [WEB-270]

#### Screenshots
Screenshot from 2022-07-18 10-18-35

[WEB-270]: https://airqoteam.atlassian.net/browse/WEB-270?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by 123MwanjeMike — APPROVED** [review]

File: website/backend/highlights/models.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from django.db import models
+from django_extensions.db.models import TimeStampedModel
+from cloudinary.models import CloudinaryField
+from author.decorators import with_author
+
+# Create your models here.
+@with_author
+class Tag(TimeStampedModel):
```
Comment (ja-odur) [inline website/backend/highlights/models.py:8]: Use the `BaseModel` from `backend.utils.model`

File: website/backend/highlights/models.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    name = models.CharField(max_length=20, null=False, blank=False)
+    created_at = models.DateTimeField(auto_now_add=True)
```
Comment (ja-odur) [inline website/backend/highlights/models.py:10]: When inheriting from the `BaseModel` or `TimeStampedModel`, the created_at and updated_at fields are auto added for you.

File: website/backend/highlights/models.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    updated_at = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        return self.name
+
+@with_author
+class Highlight(TimeStampedModel):
```
Comment (ja-odur) [inline website/backend/highlights/models.py:17]: Use BaseModel here too

File: website/backend/highlights/models.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    title = models.CharField(max_length=200)
+    tags = models.ManyToManyField(Tag)
+    image = CloudinaryField("Image", overwrite=True, resource_type="image")
+    link = models.URLField()
+    created_at = models.DateTimeField(auto_now_add=True)
```
Comment (ja-odur) [inline website/backend/highlights/models.py:22]: THese fields are auto added

**Review by ja-odur — CHANGES_REQUESTED** [review]
Left a few comments. On top of that, please delete the static files. Both collected statics and frontend static.

**Review by ja-odur — CHANGES_REQUESTED** [review]
@FaithDaka Why are css files deleted (frontend/styles/lib)? These are imported in the scss files

**Comment by FaithDaka** [comment]
Were the css files under a static folder? Because everything in a “static” directory was removed from this branch’s index. That’s all I specified in the gitignore file.

**Comment by ja-odur** [comment]
Check the changed files. The deleted scss files are from `frontend/styles/lib/ directory. They are critical scss files.

**Comment by FaithDaka** [comment]
@ja-odur **Line 17** in the _.gitignore_  file ignores all files in a lib/ directory, so reinforcing it must have deleted them from this branch to stop being tracked. But they are not deleted from my local machine.
Changing the directory name would mean changing it wherever it appears, so what should be done?

**Pull request merged**